docs/getting-started/projects/README.md

```diff
@@ -67,9 +67,9 @@ Also, you can click the green button **+ Project** directly on the system's home
 
 ![image-20200921122508132](image-20200921122508132.png)
 
-Now, you can get started working on your project.
+Creating a new project will set you as project admin, regardless of whether you copied a project, used a template or created a project from scratch. Now, you can get started working on your project.
```

Comment:
That is the default setting but may be changed by some users. We could write "By default, ..." to clarify this. (For most users the default setting will suffice but some may need to adjust this.)